packages/amazon-ssm-agent/amazon-ssm-agent.spec

```diff
@@ -76,15 +144,33 @@ ln -sf %{version} %{buildroot}%{_cross_fips_libexecdir}/amazon-ssm-agent/bin/lat
 %license LICENSE
 %{_cross_attribution_file}
 %{_cross_attribution_vendor_dir}
+%{_cross_unitdir}/amazon-ssm-agent.service
+%{_cross_tmpfilesdir}/amazon-ssm-agent-tmpfiles.conf
+%dir %{_cross_factorydir}%{_cross_sysconfdir}/amazon/ssm
+%{_cross_factorydir}%{_cross_sysconfdir}/amazon/ssm/amazon-ssm-agent.json
 
 %files bin
+%dir %{_cross_bindir}/amazon-ssm-agent
+%{_cross_bindir}/amazon-ssm-agent/amazon-ssm-agent
+%{_cross_bindir}/amazon-ssm-agent/ssm-agent-worker
+%{_cross_bindir}/amazon-ssm-agent/ssm-session-worker
```

Comment:
I see. Originally, I was installing straight to `/usr/bin` instead of its own directory. But while testing, I found that if I did this, I was not able to SSM into the instance.

However, this was with host-containers enabled, which we determined later is the incorrect setup, so let me try again.

Reply:
What Ben meant is that in the `ecs-agent.spec` file, we currently depend on the default package `amazon-ssm-agent`.

https://github.com/bottlerocket-os/bottlerocket-core-kit/blob/e4f9f14c7701338ba4a6a9ee8b1cbf55d57cefca/packages/ecs-agent/ecs-agent.spec#L86

You should update that dependency and use `amazon-ssm-agent-plugin` to pull in the correct dependency.

Reply:
I see, I can make that change in this PR. In that case, would we still need to update the ECS variants in Bob?